src/global/global_helpers.R

```diff
@@ -700,6 +701,16 @@ ms_read_raw_csv <- function(filepath,
     ){
         stop('if is_sensor is not length 1, all elements must be named.')
     }
+    
+    if(!all(data_cols %in% ms_vars$variable_code)) {
+        
+        for(i in 1:length(data_cols)) {
+            if(!data_cols[i] %in% ms_vars$variable_code) {
+                loginfo(msg = paste(unname(data_cols[i]), 'is not in varibles.csv; add'),
```

Comment:
I know you asked about this. I'm changing my answer now that I see the context. For cases like this, where the problem you're catching requires attention, we should throw an error with `stop()` instead of logging a message.